antarest/study/storage/variantstudy/repository.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from typing import Optional, List
+
+from sqlalchemy import exists  # type: ignore
+
+from antarest.core.utils.fastapi_sqlalchemy import db  # type: ignore
+from antarest.study.repository import StudyMetadataRepository
+from antarest.study.storage.variantstudy.db.dbmodel import CommandBlock
+
+logger = logging.getLogger(__name__)
+
+
+class VariantStudyCommandRepository(StudyMetadataRepository):
+    """
+    Database connector to manage Study entity
+    """
+
+    @staticmethod
```

Comment:
this is true, that (the IDE says it..) this should be static, but to be consistent with other repository (that have a __init__ constructor), this should still be a non-static method  (for now)

Reply:
same for other methods